server/services/v1/search_indexer.go

```diff
@@ -273,15 +273,6 @@ func UnpackSearchFields(doc map[string]interface{}, collection *schema.DefaultCo
 	return searchKey, tableData, doc, nil
 }
 
-func UnpackAndSetMD(doc map[string]interface{}, tableData *internal.TableData) {
```

Comment:
We don't need it now? Make sure read queries are still returning metadata along with the results.

Reply:
Working as expected:
```
{
    "result": {
        "data": {
            "review": {
                "user": "rachel",
                "votes": {
                    "downvotes": 4,
                    "upvotes": 23
                }
            },
            "brand": "Adidas",
            "description": "Performance running shoe",
            "launched": "2022-11-12T20:00:57+02:00",
            "price": 129,
            "shoe_id": "shoe_106",
            "title": "Ultraboost run - 53.0"
        },
        "metadata": {
            "created_at": "2022-08-23T00:36:06.127009024Z"
        },
        "resume_token": "c2hvZV8xMDY="
    }
}
```

Reply:
cool